### DISCUSSION TITLE
搭建archery的机器配置有最低要求吗？

### DISCUSSION THREAD

**Discussion by a8159236** [hhyo/Archery#2258] [General]
搭建archery的机器有最低配置要求吗？cpu和内存至少需要多少合适？

**Comment by LeoQuote** [comment]
没有相关建议, 我这边是 2c 2G , 大概用户是 30-50, 平均一天一个工单, 并行执行最多三个工单, 供你参考.

**Reply by a8159236** [reply]
好的谢谢
